pkgs/development/python-modules/weboob/default.nix

```diff
@@ -40,6 +40,8 @@ in buildPythonPackage rec {
     }; p' weboob/browser/browsers.py weboob/browser/pages.py
   '';
 
```

Comment:
Can you add comment like?
```code
# Would fail with `Could not find executable: pyuic5`.
# That executable will be looked up from an environment variable
# in this format when available
# 
# See: https://git.weboob.org/weboob/weboob/blob/1.3/setup.py#L31
```